arangograph/backups.md

```diff
@@ -66,17 +112,23 @@ During restore the deployment is temporarily not available.
 
 {% hint 'info' %}
 The cloned deployment will have the exact same features as the previous
-deployment including node size, model, cloud provider & region. The data
-contained in the backup will be restored to this new deployment.
+deployment including node size, model, and cloud provider. The region
+can stay the same or you can select a different one. 
+The data contained in the backup will be restored to this new deployment.
```

Comment:
For restoring as quick as possible, we recommend to create a deployment in the same region as where the backup resides, to avoid data transfer cross regions.